catalog/transformer/catalog-transformer-attribute/pom.xml

```diff
@@ -19,7 +19,6 @@
         <artifactId>transformer</artifactId>
         <version>2.14.0-SNAPSHOT</version>
     </parent>
-    <groupId>ddf.catalog.transformer</groupId>
```

Comment:
❓ Why remove the group id? It is clearer if we keep it. This applies to all subsequent pom files.

Reply:
Definition of groupId is redundant, because it's inherited from the parent

Reply:
I know it is. However, the fact is that it is much clearer if it is present.

Reply:
Inheriting the groupId follows the same paradigm as all the other pom properties we inherit and has the same benefits, like improved maintainability. 

Are you ok with adding `<!--suppress MavenRedundantGroupId -->` if we want to keep these groupIds?

Reply:
I wouldn't. It doesn't provide any valid info. I have the same comment about all the other comments you added. I feel this PR is doing a lot more than intended and some of that should probably be discussed first.

Although it is inherited, when you are looking for the group id, you have to manually traverse all in search of it. In addition! Someone might not realize that changing the group id on a pom would now affect all children poms. Properties are typically defined only in the top pom and should never be defined in other poms. It just keep maintenance more complex and makes it easier to do mistakes.